src/components/Notifications/NotificationGroupCard/NotificationGroupCard.tsx

```diff
@@ -0,0 +1,105 @@
+import { useCallback, useEffect, useState } from 'react'
+import { NotificationType } from '@dcl/schemas'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { objectToSnake, toCamel } from 'ts-case-convert'
+import { Alert, Skeleton, Snackbar, Switch } from 'decentraland-ui2'
+import NotificationIcon from '../../NotificationIcon'
+import {
+  AccordingDescription,
+  AccordingTitle,
+  AccordionDetailsStyled,
+  AccordionStyled,
+  AccordionSummaryContainer,
+  AccordionSummaryStyled,
+  NotificationItemContainer,
+  NotificationItemText,
+  NotificationItemTextIconContainer
+} from './NotificationGroupCard.styled'
+import { Props } from './NotificationGroupCard.types'
+
+function NotificationGroupCard(props: Props) {
+  const {
+    isLoading,
+    isSavingSubscription,
+    subscriptionGroupKeys,
+    notificationTypesInGroup,
+    defaultExpanded,
+    disabled,
+    subscriptionDetails,
+    error,
+    onChangeNotificationSetting
+  } = props
+
+  const [showError, setShowError] = useState(false)
+
+  useEffect(() => {
+    if (error) {
+      setShowError(true)
+    }
+  }, [error])
+
+  const handleOnChangeNotificationSetting = useCallback(
+    (checked: boolean, type: NotificationType) => {
+      if (!isLoading) {
+        const subscriptionDetailsChanged = {
+          ...subscriptionDetails,
+          messageType: { ...subscriptionDetails.messageType, [toCamel(type)]: { inApp: true, email: checked } }
+        }
+        onChangeNotificationSetting(objectToSnake(subscriptionDetailsChanged))
+      }
+    },
+    [subscriptionDetails, onChangeNotificationSetting, isLoading]
+  )
+
+  const handleClose = useCallback(() => {
+    setShowError(false)
+  }, [])
```

Comment:
If you use the local storage to not show the error, as the error is not cleared in the state, it will continue to be set and you'll see it every time you see this component.
You'll need to add a new action, to clear the error, that will be called on this handleClose method.